Machine Learning/Outlier Detection/Introduction/advent-outliers.ipynb

```diff
@@ -0,0 +1,406 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 164,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from elasticsearch import Elasticsearch, helpers\n",
+    "import requests\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib\n",
+    "from sklearn.datasets import make_blobs\n",
+    "plt.style.use('default')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Outlier Detection by Example"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Outlier detection has been available in machine learning since 7.2 - what follow is a demonstration about how to create outlier detection analyses and how to analyze the results.\n",
```

Comment:
Minor minor thing - `what follow` should be `what follows`

Reply:
thanks, fixed https://github.com/elastic/examples/pull/281/commits/8cff86bce1a05d8bcf1239aab8033490b7c76f2d